### PR TITLE
Feature api endpoints

### PR DESCRIPTION
**Pull request by askarovit**

### CODE REVIEW

File: api/service/transaction.ts [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+    getOne(data: string): ResponseEntity<TransactionModel<IAsset>>;
     getTrsByBlockId(blockId: number, limit: number, offset: number);
-
+    createTransaction(trs: TransactionModel<IAsset>): ResponseEntity<void>;
 }
 
 class TransactionService implements ITransactionService {
 
-    createTransaction(data: any) {}
+    createTransaction(trs: TransactionModel<IAsset>): ResponseEntity<void> {
```
Comment (Verbalist) [inline api/service/transaction.ts]: useless

File: core/api/controller/transaction.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { messageON } from 'shared/util/bus';
+import { API } from 'core/api/util/decorators';
+import { Message } from 'shared/model/message';
+
+export class TransactionController {
+
+    @API('TRANSACTION_CREATE')
+    public transactionCreate(message: Message): void {
```
Comment (Verbalist) [inline core/api/controller/transaction.ts]: implemented in transaction controller in core

Comment (askarovit) [reply core/api/controller/transaction.ts]: Here is the future validation before add to rxjs in the core

File: api/service/transaction.ts (hunk 1 above)
Comment (askarovit) [reply api/service/transaction.ts]: I agreee

**Comment by Verbalist** [comment]
So close, but not in time. Same code in core now

**Pull request closed without merging**